spec/fountain.spec.ts

```diff
@@ -27,7 +27,7 @@ describe('Fountain Markup Parser', () => {
             ]
         };
 
-        expect(expected).toEqual(actual);
+        expect(actual).toEqual(expected);
```

Comment:
Sorry to nitpick; I noticed they looked flipped when I was working through test results and peeked at the method definitions to confirm that the expected inputs are flipped.